src/components/actions/ActionBar.tsx

```diff
@@ -85,16 +91,20 @@ export class ActionBar extends React.Component<IActionBarProps, any> {
         : <SecondaryActions moreLabel={this.props.moreLabel} actions={secondaryActions} />;
     }
 
-    let actions = primaryActions.length || secondaryActionsView || this.props.prompt
+    const actions = primaryActions.length || secondaryActionsView || this.props.prompt
       ? (<div className='coveo-table-actions'>
         {primaryActions}
         {secondaryActionsView}
         {this.props.prompt}
       </div>)
       : null;
 
+    const containerClasses = _.isUndefined(this.props.removeDefaultContainerClasses) || !this.props.removeDefaultContainerClasses
+      ? [...ActionBar.DEFAULT_ACTIONS_CONTAINER_CLASSES, ...(this.props.extraContainerClasses || [])].join(' ')
+      : (this.props.extraContainerClasses || []).join(' ');
```

Comment:
Don't we classnames to manage classes?

Reply:
i started with it but was more verbose than this solution so reverted to that ⬆️ , having the default classes as a static variable also facilitates testing

Reply:
We can use a props classes and add a default props with DEFAULT_ACTIONS_CONTAINER_CLASSES. If the user want to override the class, he use the classes prob and by default we use the default.
I think its a lot of complexity to only manage classes with a boolean props and an array.

Reply:
the change was meant to be backward compatible, so implemented a new feature without removing the old way of using the component. It seems like what you describe would change the way it was priorly used.

Reply:
so it needs to be --> container classes by default --> use a custom mode if it does not work for you

Reply:
nvm i get what you mean, will see what I can do, might be simpler your way